assets/src/components/has-minimum-dimensions.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Gets whether the media item has the minimum dimensions.
+ *
+ * The 3 minimum dimensions for the AMP story poster image are:
+ * 696px x 928px, 928px x 696px, or 928px x 928px.
+ *
+ * @param {Object} media A media object with width and height values.
+ * @return {boolean} Whether the media has the minimum dimensions.
+ */
+export default function( media ) {
+	const largeDimension = 928;
+	const smallDimension = 696;
```

Comment:
I think this component is overly generic in its name. It is specifically about the AMP Story poster image, due to its hard-coded values here. So either the component should be renamed the required dimensions should be supplied as a second argument, e.g.:

```js
hasMinimumDimensions( media.media_details, { smallDimension: 696, largeDimension: 923  } )
```

Reply:
This component would then be able to be re-used for general validation of the selected featured image for non-story posts as detailed in https://github.com/ampproject/amp-wp/issues/1812. In fact, if you want to extend this PR to also implement a general warning for post featured image it would probably be a good idea, as it would ensure the right abstraction.

Reply:
Sounds good, Weston! I'll also extend this PR.

Reply:
Hi Weston, https://github.com/ampproject/amp-wp/pull/1998/commits/22b0f54e64952c26345920bc8dfbc79b57fddb85 prepares `hasMinimumDimensions()` for use in a different context, like #1812. But I still need to extend this PR to address #1812.